pkgs/by-name/ob/objfw/package.nix

```diff
@@ -3,19 +3,18 @@
   autogen,
   automake,
   clangStdenv,
-  fetchFromGitHub,
+  fetchfossil,
   lib,
 }:
 
 clangStdenv.mkDerivation (finalAttrs: {
   pname = "objfw";
-  version = "1.2";
+  version = "1.2.1";
 
-  src = fetchFromGitHub {
-    owner = "ObjFW";
-    repo = "ObjFW";
-    rev = "refs/tags/1.2-release";
-    hash = "sha256-nGajiLYwkIDyJujN2zWULkQXKV2A2wzjYl9IoZBU/N4=";
+  src = fetchfossil {
+    url = "https://objfw.nil.im/home";
+    rev = "559577af46b079d5d096971f35d0509c2900cd1d83cbb4a44582192481b18b6f";
```

Comment:
```suggestion
    rev = "${finalAttrs.version}-release";
```
Does this work?

Reply:
Thanks, that worked!

I've updated the PR